### ISSUE TITLE
Accessibility: Multiple occurrences for misuse of <section>  (Failure WCAG 1.3.1)

### ISSUE THREAD

**Issue by RababGomaa** [canada-ca/design-system-systeme-conception#91]
[Institutional landing page - currently in beta](https://design.canada.ca/coded-layout/institutional_landing_page_guidance.html)
`<section>` tag should be used to semantically represent generic sections of the main page. Its structure starts by a heading followed by `<h>` title for the section followed by flow content e.g. `<p>`, `<ul>` etc. . The page has Multiple occurrences for the misuse of the `<section>` element. 

**Failure 1** 
The two blocks of  '[Featured hyperlink text]' : Use of `<Section>` does not represent the relationship between the image and its caption (h3) and paragraph underneath

Suggested solutions: 
Use of `<figure>` & `<figcaption>` elements. 
```<a><figure>
<figcaption>[Featured hyperlink text]</figcaption>  
<img src="xyz.jpg" alt=xx" /> 
<p>Brief description of the feature being promoted.</p>
</figure></a>
```
Notes that:  
- Valid HTML parsing requires `<figcaption>` position either after the opening `<figure>` or before closing  `</figure>` but not in the middle 
- The `<a>` in the code suggested is wrapping the `<figure>` tag. With this position screen reader will refer the link to the the `<figure>` & content inside it.
 
**Failure 2** 
Nine blocks of "[Hyperlink text] Use action verbs..." : Coding `<section>` for structuring visual layout in columns and to present blocks of content that are not generic sections of the page 

Suggested solution: 
Change of  `<section>` to `<div>`. As the opposite of `<section> `that has a semantic meaning. `<div>` is a neutral element that does not convey semantic meanings and can be used in structuring the code and creating the visual layout. e.g. 
```
<div>
<h3>[Hyperlink text]</h3>
<p>Use action verbs, or simply list keywords....</p>
</div>
```

**Failure 3** 
Use of `<Section>` element in wrapping a `<h> `with no content and block of content that is not generic section of the main in the following blocks:
-Contact [Institution] , - [Top contact task 2], -[Top contact task 3]
-Mandate, - Transparency, Reports,... etc

Suggested solution: 
Thematic grouping for the information of Contact us, About AAFC, Minister. Possibly in one section with a general heading ?

**Related HTML & WCAG resources** 
- [Section element in HTML](https://www.w3.org/TR/html52/sections.html#the-section-element) 
- [Figure element in HTML](https://www.w3.org/TR/html52/sections.html#the-section-element)
- [G115: Using semantic elements to mark up structure ](https://www.w3.org/TR/WCAG20-TECHS/G115.html)
- [F43: Failure of Success Criterion 1.3.1 due to using structural markup in a way that does not represent relationships in the content ](https://www.w3.org/TR/WCAG20-TECHS/F43.html)

**Comment by jmealing** [comment]
I modified my comment because I've just noticed you are linking to the right links for the ILP. Team is taking a look.

**Comment by jmealing** [comment]
Sections and figcaption have been updated - thank you @RubyDo
